test1.py

```diff
@@ -1,88 +1,188 @@
-from main import WengerApi
-
+import random
 
-login2 = WengerApi("bbadescu", "TryingVPN!", "Token 3a3e4cfc386b8e7c0ea6e98a922b2bda7866ff94")
+from main import WengerApi
+from pprint import pprint
+login1 = WengerApi("balbaealexandru@gmail.com","12345678abc","Token 5c4f8bef9dd4e6d72f0868971ac142cca0157431")
 
 #ex1
-
-# print("Create weight entry")
-# login2.create_weight_entry('2021-05-31','150')
-# print("Get weight entry")
-# weight_list = login2.get_req('weightentry')
-# print(weight_list)
+print("Create weight entry")
+login1.create_weight_entry('2021-05-31','150')
+print("Get weight entry")
+weight_list = login1.get_req('weightentry')
 
 
 #ex2
 
-# for i in range(7):
-#     req1 = login2.create_nutrition_plan()
-#     print(req1)
-#
-# req1 = login2.get_req('nutritionplan')
-#
-# for nutrition in req1["results"][0:3]:
-#
-#     for i in range(5):
-#         req2 = login2.create_meals_for_nutrition_plans(nutrition['id'])
-#         print(req2)
-#
-#
-# for nutrition in req1["results"][3:7]:
-#     for i in range(5):
-#         req2 = login2.create_meals_for_nutrition_plans(nutrition['id'])
-#         print(req2)
-
-
-#ex3
-
-
-#ex4
-name_of_workout = "Workout5"
-add_workout = login2.add_workout_day(name_of_workout,"This is workout5")
+days_of_week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
+for day in days_of_week:
+    req1 = login1.create_nutrition_plan(day)
+
+
+nutrition_plans = login1.get_req('nutritionplan')
+
+for nutrition in nutrition_plans.get("results"):
+    req2 = login1.create_meals_for_nutrition_plans(nutrition.get('id'))
+    print(req2)
+
+list_of_total_meals = list()
+limit = 20
+meals1 = login1.get_req('meal',0,20)
```

Comment:
Like 29 through 36 have a duplicate code structure. How could you make all these operations using the syntax only once? 
Hint: start a for loop for something.